### ISSUE TITLE
Certain touch not flipped while mirror y-axis is enabled

### ISSUE THREAD

**Issue by Arom1a** [2394425147/astrodx#146]
**Describe the bug**
While playing VERTeX (rintaro soma deconstructed remix) transcribed chart, I used mirror y-axis to practice. At cb 507 - 570, the touch at the center top position is not flipped. 
I haven't tried other charts yet. 

**To Reproduce**
1. Enable mod mirror y-axis
2. Play VERTeX (rintaro soma deconstructed remix)

**Expected behavior**
The center top touch should also be flipped

**Screenshots**
Mirror enabled
![IMG_2737](https://github.com/user-attachments/assets/dc13b7ba-6ff6-4983-a0f1-4daddf1f4695)

Mirror disabled
![IMG_2738](https://github.com/user-attachments/assets/dc1197b8-5b60-4a5d-b1ea-6c551e1927d7)

**Device specs**
 - Device: iPad Pro
 - OS: iPadOS 17.6.1
 - Adx Version: 2.0.1

**Additional context**
None

**Comment by Soneoylys** [comment]
Can you confirm the problem again with latest updates?
Thanks

**Comment by Arom1a** [comment]
Yeah I have confirmed this bug persists in the latest version (2.0.1). 

I can also reproduce this bug in Yorugao:

![yorugao_1](https://github.com/user-attachments/assets/0d4982b4-f194-466d-bc63-fc3d7051cdee)

![yorugao_2](https://github.com/user-attachments/assets/53772efe-7f7f-42c7-b382-b48532040974)

945 cb, in the end:
![yorugao_3](https://github.com/user-attachments/assets/3516d8a7-d4d7-447f-931e-1d433570b3d9)

I believe you are familiar with this chart so the non-flipped one is omitted

**Comment by 2394425147** [comment]
Should be fixed in Beta 5 Patch 2.
Sorry for such a delayed update, but thanks for the report anyway!
